### PR TITLE
Avoid duplicated toml keys

### PR DESCRIPTION
**Pull request by vugi99**
Adding more underscores to the key name to address this issue : https://github.com/nanos-world/issues/issues/497
image

### CODE REVIEW

File: NanosWorldADK.uproject [hunk 1]
```diff
@@ -3,8 +3,19 @@
 	"EngineAssociation": "5.0",
 	"Category": "",
 	"Description": "",
+	"Plugins": [
```
Comment (gtnardy) [inline NanosWorldADK.uproject]: Could you please revert this change? I didn't want it by default

**Review by gtnardy — CHANGES_REQUESTED** [review]

**Comment by vugi99** [comment]
Is that good now ?

**Comment by gtnardy** [comment]
perfect
